deployment/docker/docker-compose.yml

```diff
@@ -170,9 +169,7 @@ services:
 
   forms-flow-webapi:
     container_name: forms-flow-webapi
-    build:
-      context: ./../../forms-flow-api/
-      dockerfile: Dockerfile
+    image: formsflow/forms-flow-webapi:v5.0.0-alpha
```

Comment:
Instead of pulling bpm and web api image we should pull from context in this repo. Else running locally for dev will be facing issue.  We can  update here instead at https://github.com/AOT-Technologies/forms-flow-ai-deployment/blob/main/docker-compose/docker-compose.yml mention this in doc and this PR should fix this.

Reply:
@sumesh-aot